docs/source/arch/esp8266/getting-started/linux.rst

```diff
@@ -53,12 +46,21 @@ or 1.4.0:
    sudo chown -R [username] ./
 
    # This will take a while...
-   make VENDOR_SDK=1.5.4 STANDALONE=y
+   make STANDALONE=y
 
-The parameter ``VENDOR_SDK=1.5.4`` specifies that you will be using SDK
-version 1.5.4. Before changing this value to a newer version make sure
-to check that the SDK version that you have chosen is
-`supported <https://github.com/SmingHub/Sming#compatibility>`__.
+Here are the links for pre-compiled versions:
```

Comment:
NONOS-SDK 1.5.4 is pretty old. A newer precompiled toolchain will be better.

Reply:
We need a repo!
https://github.com/SmingHub/Sming/issues/1905#issuecomment-547830862

Reply:
@mikee47 Please accept the invite to join attachix's core team. After that you should be able to read, write, push, etc to https://github.com/attachix/sming-toolchains. We can start putting the different toolchains there.

Reply:
@slaff Are there issues with access to sminghub?

Reply:
> sminghub?

Well yes. I am just a contributor added to SmingHub/Sming repo. And cannot create a new repository under SmingHub or add you as a core contributor to SmingHub/Sming. I also do not know for sure who is the owner of SmingHub. My wild guess is that this is not @anakod but a person who does not care so much for the success of Sming or, as strange as it may sound, might be even slightly offended by it. This is just a speculation from my side but in the last years we did not get much in  terms of support from the owner of the SmingHub account.

Reply:
Should we plan to move it then?

Reply:
> Should we plan to move it then?

Well, if the problems start to mount then yes. But before that we should carefully communicate this with the community.

Reply:
Something else we might need/want to do in the near future is migrate some parts of Sming into separate repositories, e.g. some or all of Libraries, so it's a little annoying we don't have proper control over the main repo!

I'm happy to proceed using Attachix for this and the toolchains, but it does feel a little like we're be attaching a 'brand' to it. Maybe we should set up a separate github organisation specifically for Sming? All we'd need is an inspiring name to give it!

Reply:
Re. lack of support for SmingHub, it does seem odd. There is a lot of original work in Sming, but also a considerable amount which is refinement or development of other people's code and I do try to make sure that's properly credited. If anyone reading this feels slighted or under-mentioned please accept my sincere apologies.